### ISSUE TITLE
Cannot find OCR model file

### ISSUE THREAD

**Issue by jchan698** [tmbdev/ocropy#298]
I am trying to run 
`./ocropus -rpred -m en-default.pyrnn.gz book/0001/*.png`

as part of a simple tutorial but the program complains that 

> ERROR: Cannot find OCR model file:en-default.pyrnn.gz

> ERROR: Download a model and put it into:/usr/local/share/ocropus/

> ERROR: (Or override the location with OCROPUS_DATA.)

I have put  that file in both  "ocropus_venv\ocropy-master\models" and "ocropus_venv\share\ocropus" folders in both the gunzipped and ungunzipped forms and the system still complains. All other previous commands to perform binarization and segmentation passed as expected. 

* Git revision of ocropy: most recent (just dl'ed couple of days ago)
* Python verson 2.7.13
* OS: Windows 7 64-bit

Many thanks in advance!!

**Comment by zuphilip** [comment]
Did you run `./setup.py` again?

**Comment by rogerfitz** [comment]
Just had same issue. tried wget from command line and http://www.tmbdev.net/en-default.pyrnn.gz did not resolve. Tried again in browser same thing...

But then it looks like the server "woke up" and started responding instantly -- after the web browser request. Potentially took 1 minute. I've experienced this coming from using heroku free servers that go to sleep if no requests within a time window. not sure if that is the same reason why tmbdev.net is acting funny
